code/renderers/vue3/template/cli/Page.stories.js

```diff
@@ -4,21 +4,15 @@ import MyPage from './Page.vue';
 export default {
   title: 'Example/Page',
   component: MyPage,
```

Comment:
@kasperpeulen seems that you accidentally removed the `render` function am I correct?